### ISSUE TITLE
Flame Cannon Overhaul

### ISSUE THREAD

**Issue by pa1nki113r** [pa1nki113r/Project_Brutality#421]
We want to go into a different direction with how the flame cannon was implemented in the mod, and create a more unique weapon as a result instead of copying features from other mods.

We want to make a flamethrower/Acid thrower that uses similar mechanics like the current flamecannon, although with much better sprites and animations that make it stand out.

This feature will require:

- New sprites/animations
- Code refactoring and optimizations
- Updated weapon drop lists and balance changes

**Comment by ak1dd** [comment]
Yes please!  There is currently no "flamethrower" in PB and it seems like a missing finger when you have so much variety.   I want to open a door and let the inhabitants have it with belching ribbons of flames.   The current thrower feels like a slow plasma shot.
